src/InstrumentationEngine/ProfilerManager.cpp

```diff
@@ -113,14 +120,15 @@ void CProfilerManager::FinalRelease()
 
 
 HRESULT CProfilerManager::SetupProfilingEnvironment(_In_reads_(numConfigPaths) BSTR rgConfigPaths[], _In_ UINT numConfigPaths)
```

Comment:
>SetupProfilingEnvironment [](start = 26, length = 25)

I don't know if this has any InstrumentationMethod callers, but since it's part of the api surface this could be considered a breaking change.

Reply:
Perhaps update the comment in IDL to say this function is obsolete?

---
In reply to: [381589507](https://github.com/microsoft/CLRInstrumentationEngine/pull/204#discussion_r381589507) [](ancestors = 381589507)

Reply:
I can easily implement this to forward to the new SetupProfilingEnvironment (which I'll probably rename to something more descriptive, like SetupInstrumentationMethods).